files/babel.config.cjs

```diff
@@ -0,0 +1,28 @@
+const { buildMacros } = require('@embroider/macros/babel');
+
+const macros = buildMacros();
+
+module.exports = {
+  plugins: [<% if (typescript) { %>
+    ['@babel/plugin-transform-typescript', { allowDeclareFields: true }],<% } %>
```

Comment:
do we no longer need the 2 other options in the babel config?

Reply:
why have two babel configs? https://github.com/ember-cli/ember-addon-blueprint/pull/7/files#diff-428bd93e8404f73251e66489f08bac1bf0c45cbf8a4d739e6ab6d5de5459f806R7